mw-webapp/src/service/WayService.ts

```diff
@@ -0,0 +1,31 @@
+import {ref, onValue} from "firebase/database";
+import {db} from "src/firebase";
+// import {Way} from "src/model/businessModel/Way";
+import {Way as WayDTO} from "src/model/firebaseCollection/Way";
+
+export class WayService {
+
+  public static onValueFromRealTimeDb(callBack: (data: WayDTO[]) => void) {
+    onValue(ref(db, "/ways"), async (snapshot) => {
+      const waysRaw: WayDTO[] = snapshot.val();
+      if (waysRaw !== null) {
+        const waysDTO: WayDTO[] = Object.values(waysRaw);
+        // const ways = wayDTO.map(userDTOToBusinessConverter);
+        callBack(waysDTO);
+      }
+    });
+  }
+
+  public static onValueByIdFromRealTimeDb(callBack: (data: WayDTO) => void, uuid: string) {
```

Comment:
why do you need this listener? Should both methods render triggered in case if something changed in ways

Reply:
I think it's will bw needed for edit each cell separately. But I'm bot sure right now. This task in process